bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/TypeCreator.java

```diff
@@ -248,6 +249,16 @@ public static StreamType createStreamType(String typeName, Type constraint, Modu
         return new BStreamType(typeName, constraint, modulePath);
     }
 
+    /**
+     * Create a {@code BUnionType} which represents the union type.
```

Comment:
* Create a {@code UnionType} which represents the union type. ( remove B prefix since its the impl)

Reply:
+1 fixed with 5f9a1f5